tdrs-backend/tdpservice/search_indexes/admin/ssp.py

```diff
@@ -6,6 +6,7 @@ class SSP_M1Admin(admin.ModelAdmin):
     """ModelAdmin class for parsed M1 data files."""
 
     list_display = [
+        'datafile',
```

Comment:
one tiny, tiny thing that bugged me is that this sorta makes the admin page look like the link goes to the associated datafile, but it goes to the edit page for the record

![image](https://github.com/raft-tech/TANF-app/assets/111305129/cb6d271a-31a6-4fb9-b779-6fa965801bb1)

probably because it's first in the `list_display`. making it second or last might clear it up.